api-specifications/metadata-api/Ed-Fi Version API/api/High-level metadata for the Ed-Fi API implementation-.bru

```diff
@@ -0,0 +1,67 @@
+meta {
+  name: High-level metadata for the Ed-Fi API implementation-
+  type: http
+  seq: 1
+}
+
+get {
+  url: {{baseUrl}}/api
+  body: none
+  auth: none
+}
+
+headers {
+  Accept: application/json
+}
+
+tests {
+  // Validate status 2xx 
+  //
+  // pm.test("[GET]::/api - Status code is 2xx", function () {
+  //    pm.response.to.be.success;
+  // });
+  test("[GET]::/api - Status code is 2xx", function () {
+      //pm.response.to.be.success;
+    expect(res.status).to.equal(200);
+  });
```

Comment:
The commented out code above is from Postman. Bruno automatically commented it out on import. Then I rewrote teh test using Bruno / ChaiJs syntax.